modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx

```diff
@@ -78,16 +79,125 @@ describe('View Channel', () => {
 		expect(queryByLabelText('Edit')).toBeNull();
 	});
 
+	it('should check if DELETE and CLEAR DATA buttons are displayed', () => {
+		API.user.fetchCurrentUser.mockReturnValueOnce(
+			Promise.resolve(data.mockUser())
+		);
+
+		const {queryByText} = render(<DefaultComponent />);
+
+		jest.runAllTimers();
+
+		expect(queryByText('Delete')).toBeInTheDocument();
+		expect(queryByText('Clear Data')).toBeInTheDocument();
+	});
+
+	it('should check error modal message and hyperlink on deleting property that has CHANNELS synced', async () => {
+		API.user.fetchCurrentUser.mockReturnValueOnce(
+			Promise.resolve(data.mockUser())
+		);
+
+		API.channels.fetch.mockReturnValueOnce(
+			Promise.resolve(
+				data.mockChannel(1, 1, {
+					commerceChannelsCount: 5,
+					groupsCount: 0
+				})
+			)
+		);
+
+		const {container} = render(<DefaultComponent />);
+
+		jest.runAllTimers();
+
+		expect(
+			screen.getByText(
+				'There are 0 Sites and 5 Channels synced to this property.'
+			)
+		).toBeInTheDocument();
+
+		const deleteBtn = screen.getByTestId('delete');
+
+		fireEvent.click(deleteBtn);
+
+		expect(
+			container.querySelector('div.modal-container')
+		).toBeInTheDocument();
+
+		expect(container.querySelector('div.modal-container')).toHaveClass(
+			'show'
+		);
```

Comment:
Hey @interaminense, could you put an assert to check if the text of the modal is correct? This verification is [part of the test case](https://liferay.atlassian.net/wiki/spaces/ENGAC/pages/2456355170/Test+Plan+LRAC-14082+Enable+Property+Deletion+from+DXP#Check-error-modal-message-and-hyperlink-on-deleting-property-that-has-sites-or-channels-synced---P3). Thanks